src/main/filter/abstractFilter.ts

```diff
@@ -28,7 +28,7 @@ export abstract class AbstractFilter {
 
         // Set listeners
         quickPick.onDidChangeSelection(items => {
-            this._choice = items;
+            items.forEach(item => this._choice?.push(item));
```

Comment:
I think we can do this:
```suggestion
            this._choice?.push(...items);
```